xnat-data-structure-requirements-specification.md

```diff
@@ -4,20 +4,80 @@
 
 ...
 
-**Goal**: give a structured description and insight in how non-raw DICOM and non-DICOM data (e.g., converted DICOM, segmentations, derived data) in a project is structured, what the files contain, and allow users to expose this structure through the API to be used with tooling like [xnatpy](https://xnat.rtfd.io). 
+**Goal**: give a structured description and insight in how non-raw DICOM
```

Comment:
I have the feeling that we should not limit ourselves to non-raw DICOM data, all data meant for usage stored in a project could be described by this data structure definition. It will become the single source of truth of the projects (imaging) data structure.

Reply:
agree, I think this could also for raw/original DICOM data, even though it might be simpler there because XNAT already imposes a structure here (but not a naming scheme).

Reply:
Tried to rewrite that entire goal paragraph, but I struggled a bit